### PR TITLE
Sample bundle for Jammy yoga & Focal Yoga

### PR DESCRIPTION
**Pull request by meenusachan**

### CODE REVIEW

**Review by nasharatm — APPROVED** [review]

File: juju-charms/sample_overlay_bundles/focal-yoga.yaml [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    to:
+      - '5'
+relations:
+  # Triliovault horizon plugin relations
+  # # relation with openstack-dashboard
+  - - trilio-horizon-plugin:dashboard-plugin
+    - openstack-dashboard:dashboard-plugin
+
+  # Triliovault Datamover-API relations
+  # # relation with kestone
```
Comment (abhijeetpatra) [inline juju-charms/sample_overlay_bundles/focal-yoga.yaml:71]: spell check: kestone -> keystone

**Review by abhijeetpatra — COMMENTED** [review]
rename the file names with backup target as well,
e.g focal-yoga.yaml -> focal-yoga-s3.yaml

File: juju-charms/sample_overlay_bundles/focal-yoga.yaml (hunk 1 above)
Comment (meenusachan) [reply juju-charms/sample_overlay_bundles/focal-yoga.yaml]: Fixed

**Comment by meenusachan** [comment]
> rename the file names with backup target as well, e.g focal-yoga.yaml -> focal-yoga-s3.yaml

Fixed

**Pull request merged**